packages/ckeditor5-table/tests/ui/colorinputview.js

```diff
@@ -195,21 +195,16 @@ describe( 'ColorInputView', () => {
 				const dropdown = view.dropdownView;
 				const colorTableView = dropdown.panelView.children.first;
 
-				dropdown.isOpen = true;
 				view.value = '#ffaaff';
+				dropdown.isOpen = true;
 
 				// Open color picker by clicking on button "color picker".
 				colorTableView.colorGridsPageView.colorPickerButtonView.fire( 'execute' );
-
-				view.value = '#123456';
+				colorTableView.colorPickerPageView.colorPickerView.fire( 'colorSelected', { color: 'hsl( 250, 100%, 83% )' } );
 
 				colorTableView.fire( 'cancel' );
 
-				/**
-				 * TODO, should be HEX format: (https://github.com/ckeditor/ckeditor5/issues/14432)
-				 */
-				expect( view.value ).to.be.equal( 'hsl( 300, 100%, 83% )' );
-				// expect( view.value ).to.be.equal( '#ffaaff' );
+				expect( view.value ).to.be.equal( '#ffaaff' );
```

Comment:
Maybe good idea could be creating similar test with `Remove Color`
<img width="181" alt="Screenshot 2023-06-26 at 13 21 45" src="https://github.com/ckeditor/ckeditor5/assets/47000454/b26eaa71-7f29-429b-a822-88f3d8738c73">